wp-includes/template-loader.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Loads the correct template based on the visitor's url
+ *
+ * This contains code that will amend wp-includes/template-loader.php in core.
+ *
+ * @package PWA
+ * @since 0.2
+ */
+
+/**
+ * Filter template_include to select the offline page.
+ *
+ * @since 0.2
+ *
+ * @param string $template Template.
+ * @return string Template to include.
+ */
+function _pwa_filter_template_include( $template ) {
+	$located_template = null;
+	if ( is_offline() ) {
+		$located_template = pwa_locate_template( array( 'offline.php', 'error.php' ), false );
```

Comment:
@westonruter Right now, we are repeating which templates are associated with `is_offline` and `is_500`.  Would it be better to centralize each and then use that mechanism (constant or function) to get the collection?

Reply:
Yes, good call. We need a `get_offline_template()` and `get_500_template()` to coincide with the other `get_*_template()` functions.

Reply:
Done in c9e6a1b52d92e1b299861430c48334f16b26cf0e